looper/models.py

```diff
@@ -441,6 +441,8 @@ def __init__(self, config_file, subproject=None,
         self.sheet = None
         self.samples = list()
 
+        self.add_sample_sheet()
```

Comment:
Yes, it is indeed idempotent (universally actually). I knew this before, and had at one point just left it in both places...

we can go back to the other way if you want. I was the proponent for moving them in, since I couldn't imagine a situation where you'd want to create a project and NOT add the sample sheet.